alacritty_terminal/src/config/window.rs

```diff
@@ -117,3 +123,25 @@ impl Dimensions {
         self.columns.0 as u32
     }
 }
+
+/// Window class hint
+#[serde(default)]
+#[derive(Deserialize, Debug, Clone, PartialEq, Eq)]
+pub struct ClassHint {
```

Comment:
If you do not care, then I'd name this struct `Class` and the fields `instance` and `general`.